utils/utils.go

```diff
@@ -302,14 +303,21 @@ func VerifyFFMpegPath(path string) error {
 	return nil
 }
 
-// CleanupDirectory removes the directory and makes it fresh again. Throws fatal error on failure.
+// CleanupDirectory removes all contents within the directory, or creates it if it does not exist. Throws fatal error on failure.
 func CleanupDirectory(path string) {
 	log.Traceln("Cleaning", path)
-	if err := os.RemoveAll(path); err != nil {
-		log.Fatalln("Unable to remove directory. Please check the ownership and permissions", err)
-	}
 	if err := os.MkdirAll(path, 0o750); err != nil {
-		log.Fatalln("Unable to create directory. Please check the ownership and permissions", err)
+	    log.Fatalln("Unable to create directory. Please check the ownership and permissions", err)
+	}
+	entries, err := ioutil.ReadDir(path)
+	if err != nil {
+	    log.Fatalln("Unable to read contents of directory. Please check the ownership and permissions", err)
+	}
+	for _, entry := range entries {
+	    entryPath := filepath.Join(path, entry.Name())
+	    if err := os.RemoveAll(entryPath); err != nil {
+	        log.Fatalln("Unable to remove directory. Please check the ownership and permissions", err)
```

Comment:
This error message should probably be updated to clarify that it's unable to remove the contents of the directory, and not the directory itself.